app.json

```diff
@@ -1,9 +1,16 @@
 {
   "name": "IPew Attack Map",
-  "description": "Now you can have your very own IP attack map that's just as useful as everyone else's.",
-  "repository": "https://github.com/hrbrmstr/pewpew.git",
-  "env": {
-    "BUILDPACK_URL": "https://github.com/pearkes/heroku-buildpack-static"
-  },
-  "keywords": ["security", "map", "pew"]
+  "description": "Now you can have your very own IP attack map",
+  "keywords": [
+    "security",
+    "map",
+    "pew"
+  ],
+  "repository": "https://github.com/hydrosquall/pewpew.git",
```

Comment:
I am leaving this item in just so that you can test that the new deploy button works, but this line should be changed to 

```
"repository": "https://github.com/hrbrmstr/pewpew.git",
```

before merging.

Reply:
Thank you! I shall review it in a bit!